cookbooks/omnibus-supermarket/templates/default/external.nginx.conf.erb

```diff
@@ -0,0 +1,48 @@
+# Nginx configuration for Supermarket's Rails app
```

Comment:
Is some of this common with the existing nginx conf and could it be made into an include or a partial template to make it more maintainable?